apps/prairielearn/src/tests/gradingMethods.test.js

```diff
@@ -269,16 +269,14 @@ describe('Grading method(s)', function () {
           $hm1Body = cheerio.load(hm1Body);
           iqUrl =
             siteUrl +
-            $hm1Body('a:contains("HW9.3. External Grading: Fibonacci function, file upload")').attr(
-              'href',
-            );
+            $hm1Body('a:contains("HW9.3. External Grading: Alpine Linux smoke test")').attr('href');
           questionsPage = await (await fetch(iqUrl)).text();
           $questionsPage = cheerio.load(questionsPage);
           assert.lengthOf($questionsPage('button[value="grade"]'), 1);
         });
         it('should submit "grade" action', async () => {
           gradeRes = await saveOrGrade(iqUrl, {}, 'grade', [
-            { name: 'fib.py', contents: Buffer.from(fibonacciSolution).toString('base64') },
```

Comment:
I'm surprised lint didn't pick it up, but I believe `fibonacciSolution` is now unused after this change.